opal/mca/btl/usnic/btl_usnic_mca.c

```diff
@@ -189,7 +189,7 @@ int opal_btl_usnic_component_register(void)
 
     CHECK(reg_string("if_exclude",
                      "Comma-delimited list of usNIC devices/networks to be excluded (empty value means to not exclude any usNICs).  Mutually exclusive with btl_usnic_if_include.",
-                     "lo", &mca_btl_usnic_component.if_exclude,
+                     NULL, &mca_btl_usnic_component.if_exclude,
```

Comment:
I think we do want the default value to be 127.0.0.0/8, just like the TCP BTL.

Let's punt on the other "bad" interfaces (vibr, esppp, ...etc.).  Excluding loopback interfaces by default should be fine.